### DISCUSSION TITLE
Use `<html dir="rtl">` to set direction instead of using css

### DISCUSSION THREAD

**Discussion by zheeeng** [ant-design/ant-design#37770] [Ideas]
Reference: https://www.w3.org/International/questions/qa-html-dir

> Never use CSS to apply the base direction. But do use logical ('end' and 'start') on properties or values related to margins, padding, alignment, etc. to make it easy to manage direction changes during localisation. Avoid HTML attributes with values of 'right' and 'left'.

**Comment by MadCcc** [comment]
We have adopt this feature in v5.
